plugin/evm/imports_test.go

```diff
@@ -54,10 +56,12 @@ func TestMustNotImport(t *testing.T) {
 		// Importing these packages configures libevm globally and it is not
 		// possible to do so for both coreth and subnet-evm, where the client may
 		// wish to connect to multiple chains.
-		"plugin/evm/atomic": {"core", "core/vm", "params"},
-		"plugin/evm/client": {"core", "core/vm", "params"},
-		"plugin/evm/config": {"core", "core/vm", "params"},
+		"plugin/evm/atomic": {"core", "core/vm", "plugin/evm/customtypes", "params"},
+		"plugin/evm/client": {"core", "core/vm", "plugin/evm/customtypes", "params"},
+		"plugin/evm/config": {"core", "core/vm", "params", "plugin/evm/customtypes"},
```

Comment:
nit we can remove core/vm, there is no go file there anymore

```suggestion
		"plugin/evm/atomic": {"core", "plugin/evm/customtypes", "params"},
		"plugin/evm/client": {"core", "plugin/evm/customtypes", "params"},
		"plugin/evm/config": {"core", "params", "plugin/evm/customtypes"},
```